scripts/v1alpha2/deploy.sh

```diff
@@ -33,6 +33,8 @@ kubectl apply -f manifests/v1alpha2/katib-controller/metricsControllerConfigMap.
 kubectl apply -f manifests/v1alpha2/katib-controller/katib-controller.yaml
 kubectl apply -f manifests/v1alpha2/katib/manager/service.yaml
 kubectl apply -f manifests/v1alpha2/katib/manager/deployment.yaml
+kubectl apply -f manifests/v1alpha2/katib/manager-rest/service.yaml
+kubectl apply -f manifests/v1alpha2/katib/manager-rest/deployment.yaml
```

Comment:
I think you need use new build image to replace pre-defined one in manifests/v1alpha2/katib/manager-rest/deployment.yaml in run-tests.sh to make test pass